scala-api/pom.xml

```diff
@@ -46,6 +46,19 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>com.cloudera.livy</groupId>
```

Comment:
It is Maven's requirement, one should depend on test jar to use test codes (https://maven.apache.org/plugins/maven-jar-plugin/examples/create-test-jar.html).